query-engine/connector-test-kit/src/test/scala/writes/relations/RequiredRelationSpec.scala

```diff
@@ -61,8 +61,8 @@ class RequiredRelationSpec extends FlatSpec with Matchers with ApiSpecBase {
         | }
       """,
       project,
-      errorCode = 2012,
-      errorContains = "Missing a required value at `Mutation.updateList.data.ListUpdateInput.todo"
+      errorCode = 2009,
+      errorContains = "`Mutation.updateList.data.ListUpdateInput.todo`: A value is required but not set."
```

Comment:
same like the other comment (just marking every occurence to easily spot all the changed behavior for others)